src/test/cbmc/DynTrait/main.rs

```diff
@@ -24,7 +24,11 @@ impl Animal for Cow {
 
 // Returns some struct that implements Animal, but we don't know which one at compile time.
 fn random_animal(random_number: i64) -> Box<dyn Animal> {
-    if random_number < 5 { Box::new(Sheep {}) } else { Box::new(Cow {}) }
+    if random_number < 5 {
```

Comment:
I can't put this comment on the right line since its not changed in the PR. Why not store the "random number" below, and make sure that we did the right thing based on it.